src/components/Menu/Menu/Menu.tsx

```diff
@@ -70,27 +70,15 @@ const Menu: VibeComponent<MenuProps> & {
     },
     forwardedRef
   ) => {
+    const overrideId = useMemo(() => id || `menu-${crypto.randomUUID()}`, [id]);
```

Comment:
Not sure about this one `crypto.randomUUID()`, mabe to use uuid package? Or better to use some custom function not to increase the bundle size?